unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Entries/FriendEntryBase.cs

```diff
@@ -86,8 +87,6 @@ public void OnPointerExit(PointerEventData eventData)
     protected virtual void OnDisable()
     {
         OnPointerExit(null);
-
-        model.OnTextureUpdateEvent -= OnAvatarImageChange;
```

Comment:
I think we should unsubscribe to this event somewhere. If its not OnDisable, maybe on OnDestroy?